### ISSUE TITLE
All Domain issue

### ISSUE THREAD

**Issue by bandanamanofc** [DigitalPlatDev/US.KG-Issues#89]
**Describe the bug, issue, or problem:**  
Provide a clear and concise description of the bug or problem and steps to reproduce it.

**URL (if applicable):*all .us.kg*  
Do not upload sensitive private content (such as URLs containing account information).

**Domain Name (if applicable):*all domains .us.kg*  
Example: example.us.kg

**Expected Behavior:**  Domain not found expired maybe
Describe clearly what you expected to happen.

**Screenshots:**  

**Additional Context:**  please fix it, thanks.
Add any other relevant information about the problem here.

**Comment by NotKRhero** [comment]
yeah ;< plz

**Comment by sjz123321** [comment]
yeah us.kg DNS resolve failed!

**Comment by huhao0208** [comment]
> yeah us.kg DNS resolve failed!us.kg DNS resolve failed！

+1

**Comment by OnCloud125252** [comment]
There's an [Official Announcement](https://github.com/DigitalPlatDev/FreeDomain#-announcement---december-25-2024-640-gmt0) about this issue.

TL;DR https://github.com/DigitalPlatDev/US.KG-Issues/issues/90#issue-2759365191

**Comment by EdwardLab** [comment]
Okay this is no longer a problem, I'll close
